src/QMCDrivers/DMC/DMCDriverInput.h

```diff
@@ -32,6 +32,8 @@ class DMCDriverInput
   bool get_reconfiguration() const { return reconfiguration_; }
   IndexType get_max_age() const { return max_age_; }
   IndexType get_branch_interval() const { return branch_interval_; }
+  double get_feedback() const { return feedback_; }
+  const std::string& get_refenergy_update_scheme() const { return refenergy_update_scheme_; }
```

Comment:
I was wondering where to put the enum. It is used at the input side and DMCRefEnergy. To avoid string, I need to introduce a separate header file. I initially hesitated to do that.

Reply:
changed.